### PR TITLE
feat(chat): create groups and render in sidebar

### PR DESCRIPTION
**Pull request by josephmcg**
**What this PR does** 📖
- refactor userpicker to use iridium friends list
- refactor and move GroupIcon component
  - change size to be the same as UiUserState (36px) instead of 40
  - simplify styles and html
- flesh out quickchat popup
  - remove render condition from store since its only handled in sidebar component
  - use v-model.trim to avoid needing to trim in guard clause (learn something new everyday)
- remove unused GroupClickable now that the sidebarItem component exists

**Which issue(s) this PR fixes** 🔨
AP-2002

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/interactables/UserPicker/UserPicker.vue [hunk 1]
```diff
@@ -21,17 +20,17 @@ export default Vue.extend({
...
-        if (!this.exclude.length) {
-          return all
-        }
-        return all.filter((friend) => !this.exclude.includes(friend.address))
-      },
-    }),
+    friends() {
+      if (!this.exclude.length) {
+        return this.friendsList
+      }
```
Comment (molimauro) [inline components/interactables/UserPicker/UserPicker.vue:29]: isnt this reduntant given the filter right after?

Comment (josephmcg) [reply components/interactables/UserPicker/UserPicker.vue:29]: I had the same thought when I gave my initial review lol.
Jason mentioned a performance benefit if we don't do an unnecessary filter

**Comment by josephmcg** [comment]
oh man i wrote up some longwinded explanation of everything here in the PR description but forgot to save, 1 sec

File: components/interactables/UserPicker/UserPicker.vue (hunk 1 above)
Comment (molimauro) [reply components/interactables/UserPicker/UserPicker.vue:29]: Oh okay nice!

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**